src/web/components/ArticleHeadline.tsx

```diff
@@ -211,7 +206,15 @@ export const ArticleHeadline = ({
 				case 'GuardianView':
 					return (
 						<>
-							<h1 className={cx(lightFont, invertedText)}>
+							<h1
+								className={cx(
+									lightFont,
+									invertedText,
+									css`
+										color: ${palette.headline.colour};
+									`,
```

Comment:
might be nice to have all the colours using a const like `headlineColourStyles` as its reused through the file?

Reply:
This is old code at the moment and will be removed soon (hopefully!)

Reply:
Just noticed this depends on `palette` that comes from prop, so happy with its current setup

Reply:
Yeah, we thought about that but we decided to wait until background colour comes in because that will likely trigger a larger refactor which will be easier with things inline